src/msg/async/rdma/RDMAConnectedSocketImpl.cc

```diff
@@ -58,6 +58,11 @@ RDMAConnectedSocketImpl::RDMAConnectedSocketImpl(CephContext *cct, shared_ptr<In
 {
   if (!cct->_conf->ms_async_rdma_cm) {
     qp = ib->create_queue_pair(cct, dispatcher->get_tx_cq(), dispatcher->get_rx_cq(), IBV_QPT_RC, NULL);
+    if (!qp) {
+      lderr(cct) << __func__ << "queue pair create failed" << dendl;
+      cleanup();
+      return;
```

Comment:
if we are not able to create a qp for the new connection, shall we close the accepted socket fd also, instead of using the incapable `RDMAConnectedSocketImpl` for serving the client? see `RDMAServerSocketImpl::accept()`.